plugins/jenkins/src/components/BuildWithStepsPage/BuildWithStepsPage.tsx

```diff
@@ -57,9 +55,10 @@ const BuildWithStepsView = () => {
   return (
     <div className={classes.root}>
       <Breadcrumbs aria-label="breadcrumb">
-        <Link to="../../..">Jobs</Link>
+        <Link to="../../..">Projects</Link>
         <Typography>Run</Typography>
       </Breadcrumbs>
+      <br />
```

Comment:
Probably just want to put the breadcrumbs in a wrapping div (or `<Box mb={1}>...` if you wanted some slight spacing) or so instead